src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java

```diff
@@ -146,8 +145,17 @@ protected PreSubmitCallbackResponse<SscsCaseData> issueDocument(Callback<SscsCas
 
         final String generatedFileUrl = generateFile.assemble(params);
 
-        documentTypeLabel = documentTypeLabel + ((DRAFT_DECISION_NOTICE.equals(documentType) || DRAFT_ADJOURNMENT_NOTICE.equals(documentType)) ? " generated" : " issued");
-
+        if (DocumentType.SET_ASIDE_APPLICATION.equals(documentType)) {
+            PostHearing postHearing = caseData.getPostHearing();
+            SetAsideActions action = postHearing.getSetAside().getAction();
+            if (GRANT.getValue().equals(action.getCcdDefinition())) {
```

Comment:
might be better to change this into a switch statement, will become more extendable for when we come to other post hearing journeys

Reply:
Done